test/e2e/framework/auth/helpers.go

```diff
@@ -134,19 +156,35 @@ var (
 )
 
 // IsRBACEnabled returns true if RBAC is enabled. Otherwise false.
-func IsRBACEnabled(f *Framework) bool {
+func IsRBACEnabled(crGetter v1beta1rbac.ClusterRolesGetter) bool {
 	isRBACEnabledOnce.Do(func() {
-		crs, err := f.ClientSet.RbacV1().ClusterRoles().List(metav1.ListOptions{})
+		crs, err := crGetter.ClusterRoles().List(metav1.ListOptions{})
```

Comment:
Technically we went from the `rbackV1` to `v1beta1rbac` but it is consistent with the other types throughout. I wanted to allow the callers to just pass the one object `f.ClientSet.RbacV1beta1()` rather than having to require multiple parameters.

Reply:
makes sense, seems like we should update the callers to v1 as well, but can be a follow-up